packages/components/src/submenu-popover/index.tsx

```diff
@@ -81,9 +83,9 @@ function useHasRightSpace( parentElement: HTMLElement | undefined, isVisible: bo
 }
 
 export function useSubmenuPopoverProps< T extends HTMLElement >(
-	options: { offset?: number } = {}
+	options: { offset?: number; inline?: boolean } = {}
 ) {
-	const { offset = 0 } = options;
+	const { offset = 0, inline } = options;
```

Comment:
I would use some extra params here, for every prop we use down below, in order to give the user the opportunity to pass the props he wants, and some sane defaults as well. For example:
```
options: { offset?: number; inline?: boolean; flip?: boolean; resize?: boolean; inline; boolean} = {}
```
and on initialization:

```
const { offset = 0, inline=false, flip =true, resize = true } = options;
```
Note that in case you are wondering why I put those defaults, those are what the component uses, and more specifically, those are all the default props:
```
{
		animate = true,
		headerTitle,
		onClose,
		children,
		className,
		noArrow = true,
		position,
		placement: placementProp = 'bottom-start',
		offset: offsetProp = 0,
		focusOnMount = 'firstElement',
		anchor,
		expandOnMobile,
		onFocusOutside,
		__unstableSlotName = SLOT_NAME,
		flip = true,
		resize = true,
		shift = false,
		inline = false,
		variant,

		// Deprecated props
		__unstableForcePosition,
		anchorRef,
		anchorRect,
		getAnchorRect,
		isAlternate,

		// Rest
		...contentProps
	}
```

Reply:
Thanks @kozer ! I will add those - I saw that initially Antonio passed only offset there (not sure what was the reasoning there) but makes sense! I will add those 👍